data/event-schemas/schema/m.room.avatar.yaml

```diff
@@ -10,10 +10,11 @@ properties:
           - $ref: core-event-schema/msgtype_infos/image_info.yaml
         description: Metadata about the image referred to in `url`.
       url:
-        description: The URL to the image.
+        description: |-
+          The URL to the image.
+
+          If this isn't set, that means a previous room avatar has been unset.
```

Comment:
```suggestion
          The URL to the image. If this property is not present, the room has no avatar. This can be useful
          to remove a previous room avatar.
```